samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -99,6 +100,16 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
         laughReaction: { url: 'assets/reactions/laughEmoji.png', frameCount: 102 },
         applauseReaction: { url: 'assets/reactions/clapEmoji.png', frameCount: 102 },
         surprisedReaction: { url: 'assets/reactions/surprisedEmoji.png', frameCount: 102 }
+      },
+      /* @conditional-compile-remove(on-fetch-profile) */
+      onFetchProfile: async (userId: string, defaultProfile?: Profile): Promise<Profile | undefined> => {
+        if (userId.includes('8:orgid:')) {
+          return { displayName: 'Agent' };
+        }
+        if (userId === '<28:orgid:Enter your teams app here>') {
```

Comment:
should we use environment variable instead?

Reply:
let's not hard code for sure.